samples/synchronization/sample.yaml

```diff
@@ -12,3 +12,14 @@ tests:
       regex:
         - "threadA: Hello World from (.*)!"
         - "threadB: Hello World from (.*)!"
+  sample.synchronization.openocd:
+    build_on_all: true
+    tags: kernel synchronization
+    extra_configs:
+      - CONFIG_OPENOCD_SUPPORT=y
```

Comment:
probably we want to only build this on supported architectures.

Reply:
Never mind. The problem was that I had my toolchain variant set to gnuarmemb.